docs/en/operations/settings/settings_users.md

```diff
@@ -0,0 +1,125 @@
+# User settings
+
+The `users` section of the `user.xml` configuration file aggregates settings for users.
+
+Structure of the `users` section:
+
+```
+<users>
+    <!-- If user name was not specified, 'default' user is used. -->
+    <user_name>
+        <password></password>
+        <!-- Or -->
+        <password_sha256_hex></password_sha256_hex>
+
+        <networks incl="networks" replace="replace">
+        </networks>
+
+        <profile>profile_name</profile>
+
+        <quota>default</quota>
+
+        <databases>
+            <database_name>
+                <table_name>
+                    <filter>expression</filter>
+                <table_name>
+            </database_name>
+        </databases>
+    </user_name>
+    <!-- Other users settings -->
+</users>
+```
+
+### user_name/password
+
+Password could be specified in plaintext or in SHA256 (hex format).
+
+- To specify password in plaintext (not recommended), place it in a `password` element.
+
+    For example, `<password>qwerty</password>`. Password can be empty.
+
+- To specify SHA256 hash of a password, place it in a `password_sha256_hex` element.
+
+    For example, `<password_sha256_hex>65e84be33532fb784c48129675f9eff3a682b27168c0ea744b2cf58ee02337c5</password_sha256_hex>`.
+
+    To generate password, in the command prompt perform the command:
+
+    ```
+    PASSWORD=$(base64 < /dev/urandom | head -c8); echo "$PASSWORD"; echo -n "$PASSWORD" | sha256sum | tr -d '-'
+    ```
+
+    The first line of the result is the password. The second line is the corresponding SHA256 hash.
+
+
+### user_name/networks
+
+List of networks which the user can connect to ClickHouse server from.
+
+Each element of list has one of the following forms:
+
+- `<ip>` — IP-address or a network mask.
+
+    Examples: `213.180.204.3`, `10.0.0.1/8`, `10.0.0.1/255.255.255.0`, `2a02:6b8::3`, `2a02:6b8::3/64`, `2a02:6b8::3/ffff:ffff:ffff:ffff::`.
+
+- `<host>` — Hostname.
+
+    Example: `server01.yandex.ru`.
+
+    To check access, DNS query is performed, and all received addresses compared to peer address.
+
+- `<host_regexp>` — Regular expression for host names.
+
+    Example, `^server\d\d-\d\d-\d\.yandex\.ru$`
+
+    To check access, DNS PTR query is performed for peer address and then regexp is applied. Then, for result of PTR query, another DNS query is performed and all received addresses compared to peer address. Strongly recommended that regexp is ends with $
+
+All results of DNS requests are cached till server restart.
+
+**Examples**
+
+To open access for user from any network, specify:
```

Comment:
should be accompanied by a warning that says that it's dangerous unless you have a firewall properly configured or server is not directly connected to Internet